src/components/Navbar.jsx

```diff
@@ -18,4 +18,3 @@ export class Navigationbar extends Component {
 }
```

Comment:
@Ogutu-Brian please remove this file if it has no use anymore

Reply:
Hello @koechkevin ,
I have removed it